### DISCUSSION TITLE
[source-file] add fixed width file support

### DISCUSSION THREAD

**Discussion by mjgatz** [airbytehq/airbyte#34585] [Connector Ideas and Features]
**Tell us about the problem you're trying to solve**
We want to add fixed width file support to the [file source](https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors/source-file)

**Describe the solution you’d like**
---
It appears that the source file connector is using pandas reader as the driver, pandas has a fixed-width reader:
[read_fwf](https://pandas.pydata.org/docs/reference/api/pandas.read_fwf.html)

Happy to contribute here
Thanks!

**Comment by amirsharapov** [comment]
This would be a really useful feature for us at our company as we deal with fixed width files.

[a comment by an automated account is not shown]
